### DISCUSSION TITLE
New negative --style= syntax ?

### DISCUSSION THREAD

**Discussion by cassepipe** [sharkdp/bat#2217] [Ideas]
I see that bat has quite a lot of features already for styling and I only expect that list to grow so I have a small suggestion to improve it : 

Say I want the full style but I don't want to see git modifications, I have to specify each and every option and then omit the one I don't want. 

What about a `no-` or `no` syntax as for compiler warnings or vim variables. I want everything but the `grid`,  --> `nogrid` or `no-grid`

What do you think ?

I'd implement it myself but I only know C, and only started to learn C++ (of course Rust is the next step)

**Comment by Enselic** [comment]
Sound good! There is actually already a feature request for it: https://github.com/sharkdp/bat/issues/1032
